### ISSUE TITLE
Remove `AllowTcpForwarding` addition

### ISSUE THREAD

**Issue by aequasi** [Guimove/terraform-aws-bastion#32]
I assume theres a specific reason this is here? Not sure. In my situation though, it causes my ssh's to fail

```
Host bastion.myhost.com
  	Hostname bastion.myhost.com
  	User aaron
    ForwardAgent no
  	IdentityFile ~/.ssh/id_rsa

Host 10.0.1.*
    User ubuntu
    IdentityFile ~/.ssh/my_key.pem
    ProxyCommand ssh -q -W %h:%p bastion.myhost.com
```

All the things i saw online said i need to remove it

https://unix.stackexchange.com/a/58756
https://serverfault.com/questions/535412/how-to-solve-the-open-failed-administratively-prohibited-open-failed-when-us

**Comment by Guimove** [comment]
Yes this is because de TCP Forwarding is disabled, because if we don't disable it, it's possible to bypass the log of all actions.

**Comment by Guimove** [comment]
This was released in version 1.1.1
